### PR TITLE
Develop add cv-style

### PR DESCRIPTION
**Pull request by JudithGo**

### CODE REVIEW

File: staff/judith-gongora/cv-style/style.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    }
+    section.header > div {
+        width: 100%;
+        text-align: center;
+    }
+
+    section.profile{
+        display: grid;
+        grid-template-columns: 2.5% 25% 70% 2.5%;
+        grid-template-rows: 100%;
```
Comment (seand52) [inline staff/judith-gongora/cv-style/style.css]: i think you don't need this line if the row will occupy the full space

File: staff/judith-gongora/cv-style/index.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                    <div class="name"> <h1>Judith Góngora</h1><p>Full Stack developer in the future</p></div>
+                    <div>
+                    <address>
+                        Website: 
+                        <a href="http://www.google.com" title="Personal website">http://www.google.com</a>
+                        <br>
+                        Email: 
+                        <a href="mailto:google@gmail.com" title="Email">google@gmail.com</a>
```
Comment (aleixcam) [inline staff/judith-gongora/cv-style/index.html:24]: may you want to use p tags instead of br?

File: staff/judith-gongora/cv-style/index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            </div>
+            <div class="main__interest">
+                <section class="interest">
+                    <div class="title"><h2>Interest</h2></div> 
+                    <div class="flex-interest">
+                        <div class="-group"><h4>Objetive</h4>
+                        <p>Competence development in the field of information technology. Expansion of professional experience.</p>
+                        </div>
+                        <div class="-group1"> <h4>Languajes</h4>
+                        <p>Catalan - native <br>Spanish - native <br>English - elementary</p>
```
Comment (aleixcam) [inline staff/judith-gongora/cv-style/index.html]: having different paragraphs here may be a better option than breaking one with br tags

File: staff/judith-gongora/cv-style/style.css [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+}
+
+main::after{
+    content: "";
+    clear: both;
+    display: block;
+}
+
+div {
+    /* border: 1px solid #666; */
```
Comment (aleixcam) [inline staff/judith-gongora/cv-style/style.css]: I find important to clean our code before release

**Comment by bcasasus** [comment]
Please review pull request and apply the proper changes if so

**Pull request merged**